tests/lib/rules/jsx-wrap-multilines.js

```diff
@@ -155,14 +217,14 @@ ruleTester.run('jsx-wrap-multilines', rule, {
       code: DECLARATION_TERNARY_PAREN
     }, {
       code: DECLARATION_TERNARY_NO_PAREN,
-      options: [{declaration: false}]
+      options: [{condition: false}]
```

Comment:
why does this option need to change?

Reply:
This test case checked the condition in declaration. Now checking the condition is separated.

Reply:
Does that mean that "declaration: true" now doesn't check as much as it used to? That would make this a breaking change.

Reply:
Yes, it is a broken change for declarations. But now the logic of checking is better

Reply:
Is there any way we could avoid making it breaking?

Like if `condition` is absent, it uses the old behavior, but if it's present (true or false) then it uses the new behavior?

Reply:
Yes, I can do it now. But it is not very good, so I suggest to do this only as temporary solution, and separate checking condition in major release

Reply:
Yes, definitely we'd want to separate that in the next major - but for now, we'd be able to add these new options as semver-minor.